bundles/org.openhab.binding.venstarthermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -284,6 +289,9 @@
 		<item-type>String</item-type>
 		<label>System State</label>
 		<description>Current System Operating State</description>
+		<tags>
+			<tag>Status</tag>
```

Comment:
AFAIK there should be a property and point should be combined. So to what does this Status tag relate? `Mode`? 
If not sure, please remove this tag.